README.md

```diff
@@ -205,17 +181,11 @@ Example:
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+👤 **Jose Martin Pineda**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/AztkMarty)
+- Twitter: [@twitterhandle](https://twitter.com/MartinPineda)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/AztkMarty)
```

Comment:
- [ ] kindly consider adding a working link to your LinkedIn account if you have one if not consider removing it to avoid  bad user experience to the visitors of your project